app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
As noted with the .coffee sheets, you could also remove extraneous .scss sheets you are not using as a way to clean up your directory.